app/controllers/discovered_hosts_controller.rb

```diff
@@ -50,7 +51,7 @@ def destroy
 
   def edit
     @host = ::ForemanDiscovery::HostConverter.to_managed(@host, true, false) unless @host.nil?
-    @host.attributes = @host.apply_inherited_attributes(params[:host]) unless params[:host].empty?
+    @host.attributes = @host.apply_inherited_attributes(discovered_host_params) unless discovered_host_params.empty?
```

Comment:
Oh that was it and combination of broken tests. The UI test was in bad shape. Updated, ready for re-review!